pjmedia/include/pjmedia/stream_common.h

```diff
@@ -54,6 +194,89 @@ typedef struct pjmedia_stream_rtp_sess_info
 
 } pjmedia_stream_rtp_sess_info;
 
+
+/**
+ * Get the stream statistics. See also
+ * #pjmedia_stream_get_stat_jbuf()
+ *
+ * @param stream        The media stream.
+ * @param stat          Media stream statistics.
+ *
+ * @return              PJ_SUCCESS on success.
+ */
+PJ_DECL(pj_status_t)
+pjmedia_stream_common_get_stat( const pjmedia_stream_common *stream,
+                                pjmedia_rtcp_stat *stat);
+
+
+/**
+ * Reset the stream statistics.
+ *
+ * @param stream        The media stream.
+ *
+ * @return              PJ_SUCCESS on success.
+ */
+PJ_DECL(pj_status_t)
+pjmedia_stream_common_reset_stat(pjmedia_stream_common *stream);
+
+
+/**
+ * Send RTCP SDES for the media stream.
+ *
+ * @param stream        The media stream.
+ *
+ * @return              PJ_SUCCESS on success.
+ */
+PJ_DECL(pj_status_t) 
+pjmedia_stream_common_send_rtcp_sdes( pjmedia_stream_common *stream );
+
+/**
+ * Send RTCP BYE for the media stream.
+ *
+ * @param stream        The media stream.
+ *
+ * @return              PJ_SUCCESS on success.
+ */
+PJ_DECL(pj_status_t)
+pjmedia_stream_common_send_rtcp_bye( pjmedia_stream_common *stream );
+
+
+/**
+ * Get the RTP session information of the media stream. This function can be
+ * useful for app with custom media transport to inject/filter some
+ * outgoing/incoming proprietary packets into normal audio RTP traffics.
+ * This will return the original pointer to the internal states of the stream,
+ * and generally it is not advisable for app to modify them.
+ *
+ * @param stream        The media stream.
+ *
+ * @param session_info  The stream session info.
+ *
+ * @return              PJ_SUCCESS on success.
+ */
+PJ_DECL(pj_status_t)
+pjmedia_stream_common_get_rtp_session_info(pjmedia_stream_common *stream,
+                                   pjmedia_stream_rtp_sess_info *session_info);
```

Comment:
Are these new APIs being used somewhere or perhaps will be in the future?

Reply:
Yes, most of the new APIs will be used by the new text stream.